src/Analysis/GWASUIApp/GWASJob.jsx

```diff
@@ -0,0 +1,175 @@
+import React from 'react';
+import {
+  Button, List, Tag,
+} from 'antd';
+import {
+  CheckCircleOutlined,
+  SyncOutlined,
+  CloseCircleOutlined,
+  QuestionCircleOutlined,
+  MinusCircleOutlined,
+  StopOutlined,
+} from '@ant-design/icons';
+import { useQuery } from 'react-query';
+import PropTypes from 'prop-types';
+import { gwasWorkflowPath } from '../../localconf';
+import { headers } from '../../configs';
+import { getPresignedUrl } from '../AnalysisJob';
+
+const GWASJob = ({ workflow }) => {
+  async function handleWorkflowOutput(url) {
+    const response = await fetch(url, { headers }).then((res) => res.json()).then((data) => data);
```

Comment:
we usually don't do this, any reason for not using the `fetchWithCreds()` from actions?

Reply:
I don't think anything from `GWASUIApp` is used anymore tbh

Reply:
the response is passed to `getPresignedUrl()` which uses `fetchWithCreds()`.